### PR TITLE
IDO Components - REKRUTACJA

### PR DESCRIPTION
**Pull request by EveScolded**
IDO components and static page

### CODE REVIEW

File: src/components/Labels/IDOLabels/IDOLabels.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  title: ReactElement | string
+  icon: string
+  value: string
+}
+export const IDOLabels: React.FC<IIDOLabelsProps> = ({ className, title, icon, value }) => {
+  const classes = useStyle()
+
+  return (
+    <div className={classNames(classes.general, className)}>
+      <label className={classes.title}>{title}</label>
```
Comment (awojciak) [inline src/components/Labels/IDOLabels/IDOLabels.tsx:17]: for texts use "Typography element" from material-ui; there is wrong font in stories for that component

**Review by awojciak — COMMENTED** [review]
also: 
1) i think there should be a story for whole IDO
2) I'm unable to write anything in input; max button is also not working; basic functionality like that should work in component

**Review by awojciak — COMMENTED** [review]
1) 
obraz
this should be wider
2) there is bottom line showing when hovering/clicking input - it shouldn't show

**Comment by EveScolded** [comment]
> also:
> 
> 1. i think there should be a story for whole IDO
> 2. I'm unable to write anything in input; max button is also not working; basic functionality like that should work in component

Input is used as controlled component. Button uses empty callback because I've only wanted to display it. The IDO page is prepared for illustration only. I'm eager to fix other remarks

**Pull request closed without merging**